src/tools/annotation/bidirectionalTool/moveHandle/moveLongLine/getDistanceToIntersection.test.js

```diff
@@ -0,0 +1,44 @@
+import getDistanceToIntersection from './getDistanceToIntersection';
+
+function createPoint(x, y) {
+  return {
+    x,
+    y,
+  };
+}
+
+describe('getDistanceToIntersection.js', () => {
+  it('distance is positive', () => {
```

Comment:
Doesn't read well. You can use `it('description')` or `test('description')`, but it should read like a simple requirement.

`it('returns a positive distance, regardless of point order')` or...
`test('returned distance is positive')`

Reply:
This function its test has been removed.
But thanks for the review.